flask/app/routes.py

```diff
@@ -162,3 +163,42 @@ def change_password():
     except:
         db.session.rollback()
         return 'Server error', 500
+
+
+@app.route('/api/participants/<participant_id>', methods = ['PATCH'])
+# @login_required # check document for how to test endpoints with login signing? 
```

Comment:
Keep this on. For your testing you can either comment this out (but don't commit it) or in config.py add `LOGIN_DISABLED = True` to disable it globally.